### ISSUE TITLE
Trying to mint NFT and getting TxMetadataJsonToplevelBadKey 

### ISSUE THREAD

**Issue by RhysGoodwin** [functionally/mantis#1]
Probably something silly I'm doing. Total newbie here. Any help much appreciated. 

mint file - (EDITED. fix copy/paste error) I have also tried with the example mint file in the readme.md: 
```
{
"sImple iSgOod": {
    "name": "sImple iSgOod"
  },
  "nOthing bUtnAme": {
    "name": "nOthing bUtnAme"
  },
  "pRopers tOwn": {
    "name": "pRopers tOwn"
  }
}
```

config file:
```
Configuration {
  socketPath          = "/var/lib/docker/volumes/servername_node-ipc/_data/node.socket"
, magic               = Just 1097911063
, epochSlots          = 21600
, addressString       = "addr_test1vz8urxkrmm9zfg8sw9wgsvfrtycywmf0zujq79cp9npef7g9kl9jv"
, verificationKeyFile = "/var/lib/docker/volumes/servername_nft-data/_data/project10/payment.vkey"
, signingKeyFile      = "/var/lib/docker/volumes/servername_nft-data/_data/project10/payment.skey"
}
```

cmd
`./mantis mint /var/lib/docker/volumes/nopow_nft-data/_data/project10/testnet.mantis /var/lib/docker/volumes/nopow_nft-data/_data/project10/mints.json --expires +200 --script /var/lib/docker/volumes/nopow_nft-data/_data/project10/policy.script --metadata /var/lib/docker/volumes/nopow_nft-data/_data/project10/metadata.json`

Output:
```
Network: Testnet (NetworkMagic {unNetworkMagic = 1097911063})

Tip: SlotNo 29040648

Protocol parameters: {"txFeePerByte":44,"minUTxOValue":1000000,"stakePoolDeposit":500000000,"decentralization":0,"poolRetireMaxEpoch":18,"extraPraosEntropy":null,"stakePoolTargetNum":500,"maxBlockBodySize":65536,"maxTxSize":16384,"treasuryCut":0.2,"minPoolCost":340000000,"maxBlockHeaderSize":1100,"protocolVersion":{"minor":0,"major":4},"txFeeFixed":155381,"stakeAddressDeposit":2000000,"monetaryExpansion":3.0e-3,"poolPledgeInfluence":0.3}

Input Address: addr_test1vz8urxkrmm9zfg8sw9wgsvfrtycywmf0zujq79cp9npef7g9kl9jv
Output Address: addr_test1vz8urxkrmm9zfg8sw9wgsvfrtycywmf0zujq79cp9npef7g9kl9jv

Verification key hash: "8fc19ac3deca24a0f0715c8831235930476d2f17240f17012cc394f9"
Signing key . . . read successfuly.

Unspect UTxO:
  Transaction: 1fe81d731f47cef612111ef65000cc9c7b17a8d89657d7b81249d82cb412d7cd#TxIx 0
    Lovelace 5000000 Lovelace

Total value:
  Lovelace 5000000 Lovelace

Policy ID: "42f57b5515431258f900ecda6a2fa144b664c4c19b06c42e485d0463"
Policy: ScriptInEra SimpleScriptV2InMary (SimpleScript SimpleScriptV2 (RequireAllOf [RequireSignature "8fc19ac3deca24a0f0715c8831235930476d2f17240f17012cc394f9",RequireTimeBefore TimeLocksInSimpleScriptV2 (SlotNo 29040848)]))
TxMetadataJsonToplevelBadKey "sImple iSgOod"
```

Thanks,
Rhys

**Comment by bwbush** [comment]
I'm looking into this now.

**Comment by bwbush** [comment]
Please accept my apologies: this was a bug in how the NFT metadata was augmented for minting. It is fixed in version 0.2.10.2, which you can reinstall.

**Comment by RhysGoodwin** [comment]
Thanks Brian! Tested all ok! Thanks so much for you help and for sharing your code.
Kind regards from Aotearoa,
Rhys
